core/src/main/java/org/sql2o/converters/BigIntegerConverter.java

```diff
@@ -0,0 +1,39 @@
+package org.sql2o.converters;
+
+import java.math.BigInteger;
+
+public class BigIntegerConverter extends NumberConverter<BigInteger> {
+
+
+    public BigIntegerConverter() {
+        super(false);
+    }
+
+    @Override
+    protected BigInteger convertNumberValue(Number number) {
+        if(null == number){
+            return null;
+        }
+        else if (number instanceof BigInteger){
+            return (BigInteger)number;
+        }
+        else{
+            return BigInteger.valueOf(number.intValue());
```

Comment:
Is seems that `number.longValue()` would be better than  `number.intValue()`. Otherwise you can lost precision.
Also, it's more native to use long than int when you want to get BigInteger